scripts/helpers.sh

```diff
@@ -41,7 +41,7 @@ tmux_socket() {
 }
 
 session_exists() {
-	tmux has-session -t "$SESSION_NAME" >/dev/null 2>&1
+	tmux has-session -t="$SESSION_NAME" >/dev/null 2>&1
```

Comment:
Hey,
I agree the exact matching is better when creating a new session. In fact I have been annoyed by this in the past, but never found time to fix.

However, this helper is also used for the `prefix + g` feature which performs a basic "session name completion + switch". Example, if session name "trunk" exist, we want to be able to switch to it by just typing "tru", or even just "t" if that's the only session name that starts with "t".

So I'm ok with this addition, but please change it so that `prefix + g` stays "fuzzy", and exact matching is used only when creating a session.